tests/test_qc_on_db.py

```diff
@@ -794,6 +794,33 @@ def test_do_sst_missing_value_check(testdata):
 
 
 def test_do_sst_freeze_check(testdata):
+    db_ = testdata["observations-sst"].copy()
+    results = do_sst_freeze_check(
+        sst=db_["observation_value"],
+        freezing_point=271.35,
+        freeze_check_n_sigma=2.0,
+    )
```

Comment:
No more pandas apply function is needed to call the QC functions 🎉.
I think this is very helpful for users.